check-markdown.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable  no-console */
+const markdownlint = require('markdownlint');
+
+const options = {
+  files: process.argv.slice(2),
```

Comment:
I think that you will loose some changed files, to properly support running this script from `lint-staged` and from npm script or directly you need to do something like this: https://github.com/auth0/auth0-website/blob/master/tools/scripts/prettier#L36-L54

You can modify `lint-staged` source to see the stdout here https://github.com/okonet/lint-staged/blob/c7283b77c2cf3637e9f06f953b919a556ffde205/src/runScript.js#L19-L20 with:
```js
// ...
.then(result => {
	console.log(result.stdout);
// ...
```